concepts/05 Widgets/DateBox/00 Getting Started with DateBox/10 Set the DateBox Type.md

```diff
@@ -0,0 +1,56 @@
+To allow users to specify the date and time, set the [type](/Documentation/ApiReference/UI_Widgets/dxDateBox/Configuration/#type) to *"datetime"*. This will implicitly set [applyValueMode](/Documentation/ApiReference/UI_Widgets/dxDateBox/Configuration/#applyValueMode) to *"useButtons"*, i.e. a user has to press OK to submit their choice. 
```

Comment:
```suggestion
To allow users to specify the date and time, set the [type](/Documentation/ApiReference/UI_Widgets/dxDateBox/Configuration/#type) to *"datetime"*. This will implicitly set [applyValueMode](/Documentation/ApiReference/UI_Widgets/dxDateBox/Configuration/#applyValueMode) to *"useButtons"* (i.e., users must press OK to submit their choices). 
```